src/browser/settings.jsx

```diff
@@ -33,8 +38,22 @@ var SettingsPage = React.createClass({
       showAddTeamForm: false
     });
 
+
     return config;
   },
+  componentDidMount: function() {
+    if (process.platform === 'win32' || process.platform === 'linux') {
+      appLauncher.isEnabled().then(function(enabled) {
+        this.setState({
```

Comment:
The context of `this` is changed by function object. To use `this` here, an arrow function should be used for this callback.